packages/engine/src/interaction/components/InteractableComponent.ts

```diff
@@ -323,7 +324,13 @@ export const InteractableComponent = defineComponent({
     InputComponent.useExecuteWithInput(
       () => {
         const buttons = InputComponent.getMergedButtons(entity)
-        if (!interactableComponent.clickInteract.value && buttons.PrimaryClick?.pressed) return
+        if (
+          !activateUI ||
```

Comment:
This is scoping a global variable from a different context. We need a different solution.

Reply:
implemented